tests/unit/result_tests.py

```diff
@@ -643,5 +645,43 @@ def test_iteration_no_data(self):
         result = Result(self.view001, startkey='ruby')
         self.assertEqual([x for x in result], [])
 
+    def test_iteration_integer_keys(self):
+        """
+        Test that iteration works as expected when keys are integer.
+        """
+        result = Result(self.view007)
```

Comment:
can you put a small page size in here? maybe `page_size=10`

Reply:
sure